docs/api/class-sections.md

```diff
@@ -6,8 +6,12 @@ The `class section` resource has two possible representations: a detailed (or fu
 ## Properties
 All properties which are not assigned with the `mandatory` label, are optional, they may not be included in the representation.
 
-* `course`: the course's unique acronym
+* `courseId`: the course's unique identifier
   - mandatory
+  - type: **number**
+  - e.g. 1
+
+* `courseAcr`: the course's unique acronym
   - type: **text**
   - e.g. "WAD"
```

Comment:
Shouldn't the properties not have relationship information, so no `courseId`, `courseAcr` and `class`? I think the only property the `ClassSection` should have is its Id. And why does it need to have a relation to `Course` if it already has one to `Class`?